packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -1397,6 +1397,8 @@ If you configured your client to use the `client_secret_jwt` client authenticati
 
 Provide the `client_id` in a JWT that you sign with the `client_secret` using an HMAC SHA algorithm (HS256, HS384, or HS512). The JWT must also contain other values, such as issuer and subject. See [Token claims for client authentication with client secret or private key JWT](/docs/reference/api/oidc/#token-claims-for-client-authentication-with-client-secret-or-private-key-jwt).
 
+> **Note:** JWTs with a shared key require a secret that is at least 32 characters in length to satisfy HS256 cryptographic minimums. Clients that attempt to set `token_endpoint_auth_method` to `client_secret_jwt` with an imported secret less than 32 characters will receive a validation error. Clients that send Okta a JWT for verification signed with HS256, HS384, or HS512 with a secret less than 32 characters will receive an error: `oauth2.error.code.client_assertion_invalid_secret_length`.
```

Comment:
`oauth2.error.code.client_assertion_invalid_secret_length` is not the error the client will receive, that is the localization string

Reply:
1. Should `HS256` , `HS384` and `HS512` be in back-ticks?
2. Do we have say "will receive a validation error", I think just expected behavior should be sufficient. It is understood that if clients do not meet those requirements, they will have a validation error. 
3. Not sure if we need to talk about both imported secrets and signatures, given both are requirements for using a `HS*` algorithm?

**Proposal**:
Client Secret JWTs using `HS256` , `HS384` or `HS512` schemes require a secret length of at least 32 characters to satisfy minimum cryptographic requirements.